src/amo/components/AddonReviewListItem/index.js

```diff
@@ -60,12 +60,20 @@ type InternalProps = {|
 
 export class AddonReviewListItemBase extends React.Component<InternalProps> {
   onClickToDeleteReview = (event: SyntheticEvent<HTMLElement>) => {
-    const { dispatch, errorHandler, isReplyToReviewId, review } = this.props;
+    const {
+      addon,
+      dispatch,
+      errorHandler,
+      isReplyToReviewId,
+      review,
+    } = this.props;
     event.preventDefault();
 
+    invariant(addon, 'addon is required');
     invariant(review, 'review is required');
     dispatch(
       deleteAddonReview({
+        addon,
         errorHandlerId: errorHandler.id,
         reviewId: review.id,
```

Comment:
I thought it would be clever to change this from `review.id` to `review` so you could use `review.addon` but that doesn't get us a `guid`. Oh, well.

https://github.com/mozilla/addons-frontend/blob/eae17b8d2dad542e97ecc00cffe5fc870554467f/src/amo/api/reviews.js#L13-L16